ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -799,8 +804,8 @@ const MultichainTransactionListItem = ({
               className="activity-tx__network-badge"
               data-testid="activity-tx-network-badge"
               size={AvatarNetworkSize.Xs}
-              name={networkConfig.id}
-              src={networkConfig.rpcPrefs?.imageUrl}
```

Comment:
Why cant we use the image from the network config anymore?

Reply:
The network configuration has changed, this is a bit of legacy code so we're slowly moving away from this data structure